Gemfile

```
@@ -92,7 +92,7 @@ gem 'mimemagic', '~> 0.3.10'
 gem 'nokogiri', '~> 1.10.10'
 gem 'secure_headers', '~> 6.3.0'
 
-gem 'acts-as-taggable-on', '~> 4.0'
+gem 'acts-as-taggable-on', '~> 8.0'
```

Any reason why not 8.1 here?
```suggestion
gem 'acts-as-taggable-on', '~> 8.1'
```

---

it doesn't matter, it picked up 8.1 anyway because the matcher means latest 8.x